CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 ## HEAD
 
+## 0.15.0-beta.2
+###### _Apr 21, 2016_
+
+##### General
+- [.gitignore] Ignore `jsconfig.json` - VSCode config file (#4011)
+- [Docs] Update usage docs with muiTheme instructions (#4034)
+- [Docs] Add beta installation details to the README (#4048)
+- [Examples] Update import statements (#3992)
+
+##### Component Fixes / Enhancements
+- [AutoComplete] Change `error`, `hint`, `floatingLabel` property validators to `PropTypes.node` (#4019)
+- [Dialog] Add border to title and actions when content is scrollable (#4001)
+- [Dialog] Add support for the Alert (#4022)
+- [Dialog] Merge title style when title it a node (#4033)
+- [ListItem] Add extra width to `leftCheckbox` for `EnhancedSwitch` margin, fixes #4016 (#4044)
```

Comment:
Yeah, we forgot to change that when the solution changed. Let's fix it in the changelog at least.